MhLabs.AwsSignedHttpClient/HttpOptions.cs

```diff
@@ -1,9 +1,17 @@
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions;
+
 namespace MhLabs.AwsSignedHttpClient
 {
     public class HttpOptions
     {
+        public HttpOptions(ILogger logger)
```

Comment:
I think its fine that we make a breaking change here.
But it is also possible to make the `logger` parameter optional and then default the Logger property to `NullLogger.Instance`

Not sure if we want to do that tho :)

Reply:
I'll add the logging parameter to top level extension. As JP said, HttpOptions has nothing to do with logging.